server/adapters/index.js

```diff
@@ -1,5 +1,7 @@
 module.exports = function initAdapters(libs, conf, models, workers) {
   const adapters = {};
 
+  require('./logAdapter')(models, adapters);
```

Comment:
If we don't add it to the adapter object, how will we get it from the libs object ? Will there be a loop that will add them all to the adapter object or do we need to make a require where we need the module ?